CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Add the ``configureActionButtons`` method to the AdminInterface
 - Add the ``configureActionButtons`` method to the AdminExtensionInterface
+- Add the ``getAccessMapping`` method to the AdminExtensionInterface
```

Comment:
Is there any difference to two ticks? Thought this is the same. The other code snippets use two ticks.